snyk/managers.py

```diff
@@ -138,31 +139,89 @@ def delete(self, key, value) -> bool:
 
 
 class ProjectManager(Manager):
-    def _query(self, tags: List[Dict[str, str]] = []):
+    def _rest_to_v1_response_format(self, project):
+        attributes = project.get("attributes", {})
+        settings = attributes.get("settings", {})
+        recurring_tests = settings.get("recurring_tests", {})
+        issue_counts = project.get("meta", {}).get("latest_issue_counts")
+
+        return {
+            "name": attributes.get("name"),
+            "id": project.get("id"),
+            "created": attributes.get("created"),
+            "origin": attributes.get("origin"),
+            "type": attributes.get("type"),
+            "readOnly": attributes.get("read_only"),
+            "testFrequency": recurring_tests.get("frequency"),
+            "isMonitored": True
+            if project.get("meta", {}).get("cli_monitored_at")
+            else False,
+            "issueCountsBySeverity": {
+                "low": issue_counts.get("low"),
+                "medium": issue_counts.get("medium"),
+                "high": issue_counts.get("high"),
+                "critical": issue_counts.get("critical"),
+            },
+            "targetReference": attributes.get("target_reference"),
+            "_tags": attributes.get("tags", []),
+            "browseUrl": f"https://app.snyk.io/org/{project.get('relationships', {}).get('organization', {}).get('data', {}).get('id')}/project/{project.get('id')}",
```

Comment:
I am assuming PySnyk only understand one base URL which is `snyk.io` and nothing else? I see URLs hardcoded in a couple of places, rather than using some parameter so just checking

Reply:
when creating a [snyk client](https://github.com/snyk-labs/pysnyk#client) with pysnyk, you can specify alternate base URLs for the API. This is probably super relevant for other tenants (MT-EU, MT-AU, single tenant instances) .. It's too bad we dont get the browser URL back as data from the API.  If customer is using another tenant I dont think the "app.snyk.io" link would work if their base API for a single tenant instance was lets say https://api.{customer}.my.snyk.io/rest, then the coresponding project link would use https://app.{customer}.my.snyk.io

If the v1 API correctly returns the right browseURL, then hard-coding this here would introduce a bug

Reply:
This is a good point.

Sadly, I don't see a way to get this data automatically, this has to be constructed from the fields that we get back from the REST API (which of course doesn't include the browse URL).

The only option here would be to check the base url that has been set, check which tenant it is and construct the base url based upon those results. It's messy, but there's a finite number of tenants and there's no other way I can see.

Reply:
I've pushed new changes on this, browseUrl is now lazy-loaded and should respect the tenant based on the `api_url` defined in the `SnykClient` instance.